### PR TITLE
per-platform noarch

### PR DESCRIPTION
**Pull request by minrk**
We can have noarch builds if they are per-platform

closes #174 
closes #175 
closes #176

This avoids the need to rebuild for Python updates, etc.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,21 +26,25 @@ build:
...
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
     - pip
-    - python
+    - python >=3.8
   run:
-    - python
-    - appnope  # [osx]
+    - __{{ noarch_platform }}
```
Comment (bollwyvl) [inline recipe/meta.yaml:33]: sick jams :guitar:

**Comment by mattip** [comment]
Cool, CI is passing

**Review by bollwyvl — APPROVED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,10 +11,10 @@ source:
...
 
 build:
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv --no-deps
 
-  skip: true  # [py<38]
+  noarch: python
   script_env:
     - MIGRATING={{ migrating }}
```
Comment (bollwyvl) [inline recipe/meta.yaml:19]: will we even need this anymore?

Comment (minrk) [reply recipe/meta.yaml:19]: I don't know what that is for, so 🤷

**Comment by Carreau** [comment]
+1 for me, but I don't feel qualified to review. Should this be merged now ?

**Pull request merged**